<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTGUI_TestClass</class>
 <widget class="QMainWindow" name="QTGUI_TestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTGUI_Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>启动服务器</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>客户端提交的数据（A,B）</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>A:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="a_text">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>B:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="b_text">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>381</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>服务端处理的结果（C= A + B）</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>C：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="c_text">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>381</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>设置最大连接数</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>M：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="c_text_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>停止服务器</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>start_button</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>stop_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtgui_test.qrc"/>
 </resources>
 <connections/>
</ui>
